index.js

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable import/extensions */
+
+import Storage, {
+  Card,
+  arrBooks,
+  bookTitle,
+  bookAuthor,
+} from './modules/storage.js';
+import Books from './modules/books.js';
+import { DateTime } from './modules/luxon.js';
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] Kindly remove the eslint-disabler as its use is not permitted. There are unused variables, bodyCont, tableContainer, and navbar, which may be causing this issue and can be removed if they are not necessary.


> Sometimes we encounter challenges when using ESLint. Unexpected errors may arise which are difficult to resolve quickly. If you find an issue that is functioning properly, but is still displaying an error message when uploaded to GitHub, I recommend disabling in `.eslintrc.json`.